### DISCUSSION TITLE
Repository Issues

### DISCUSSION THREAD

**Discussion by iLakshya** [community/community#34902] [New to GitHub] [answered]
Hey everyone,

Can someone please tell me how should we clone a repository and how should I add this to my local repo?

Happy Learning...✌🏻

**Comment by airtower-luna** [comment, marked as answer]
Check the [Git Book](https://git-scm.com/book/en/v2) website, and make sure you have Git installed. I recommend you read (at least) the first three chapters, but the things you probably need most are:

* 2.1 [Getting a Git Repository](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository) (for cloning)
* 2.2 [Recording Changes to the Repository](https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository) (for committing your files locally)
* 2.5 [Working with Remotes](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes) (to learn about pushing back to GitHub, if you want to)

**Comment by Ajit-pawara** [comment]
Hello, I would like to delete the forked repository GC.github.io from my account. Since I do not have admin rights, I cannot delete it manually. Please assist me in removing this fork. Thank you!
